constellations/gemini-admin/src/help/help.styl

```diff
@@ -0,0 +1,25 @@
+@import "../static.styl"
```

Comment:
ah sorry make this a .sass file instead of .styl. We want to move away from .styl